vg_postnord.php

```diff
@@ -953,7 +953,7 @@ public function hookActionAdminControllerSetMedia()
     /**
      * Add the CSS & JavaScript files you want to be added on the FO.
      */
-    public function hookHeader()
+    public function hookDisplayHeader()
```

Comment:
did you check if this affects our ps_min_version?

Reply:
I checked up to 1.7.6 (it doesn't): https://github.com/PrestaShop/PrestaShop/blob/368fc858b8cbc1f40949412d28ac2d273bae6abc/classes/controller/FrontController.php#L556